content/docs/dvclive/api-reference/live/set_step.md

```diff
@@ -29,18 +29,20 @@ You can use `Live.set_step()` to set `step` to any value.
 Each metric logged in between `Live.set_step()` (or `Live.next_step()`) calls
 will be associated to the provided `step` value.
 
+<admon type="info">
+
+If `report="html"` was passed to `Live()`, each `Live.next_step()` will also
+call `Live.make_report()`.
```

Comment:
I don't see this `report="html"` option/argument documented in https://dvc.org/doc/dvclive/api-reference/live . Is it the same as the `html_report` attribute?

Reply:
> I don't see this `report="html"` option

Nvmd on that. I see it's [added in this PR](https://github.com/iterative/dvc.org/pull/3411/files#diff-872989cee0bf70053eb56c875d6c3e0a76105202d216502e604b1d6c11163c2d) 😋